api/handlers.go

```diff
@@ -2482,6 +2492,19 @@ func appendTagsFilterMods(exec boil.Executor, mods *[]qm.QueryMod, f TagsFilter)
 	return nil
 }
 
+func prepareNestedTagIds(exec boil.Executor, f TagsFilter) (pq.Int64Array, pq.StringArray, error) {
```

Comment:
Maybe we don't go to the DB for this on every request and use Cache manager instead.
https://github.com/Bnei-Baruch/archive-backend/blob/27d9662d37694d4d68b39257827677993380e26c/common/common.go#L25
We might need to add some function to get what we need but search_stats basically holds the sources tree and tags hierarchy in memory. Better use that. It also refreshes every 5 minutes or so. I think it's ok to use it in here.

Reply:
I need to discuse with you about this.
Do you mean replace and [this](https://github.com/Bnei-Baruch/archive-backend/blob/master/api/filters_stats.go) ?
So Iwill need add other CacheManager imlimentation, but dont want load twice [this](https://github.com/Bnei-Baruch/archive-backend/blob/4cd765e70767f939ad342b9e4d4a7b6460da8a44/cache/search_stats.go#L407) 
So I must move out [fetch data](https://github.com/Bnei-Baruch/archive-backend/blob/4cd765e70767f939ad342b9e4d4a7b6460da8a44/cache/search_stats.go#L296) from SearchStatsCacheImpl ...
And I think we have other places where we use source and tad hierarchy
Is it good idea to push CU UIDs and C UIDs? (I dont think so but ...)
So need to discuss )